### ISSUE TITLE
inverse option is not working in ios

### ISSUE THREAD

**Issue by aparnascodex** [tomickigrzegorz/circular-progress-bar#87]
We are trying to use the library in one of our plugin, but inverse option is not working in ios. Another is sometimes the circle goes on and on, but sometimes is works fine.

**Comment by tomickigrzegorz** [comment]
As they say, "It works for me" iphone safari

https://github.com/tomickigrzegorz/circular-progress-bar/assets/25438490/9ebcca91-8344-4d77-b64e-2d816c97f675

**Comment by aparnascodex** [comment]
<img width="203" alt="Screenshot 2024-02-03 230845" src="https://github.com/tomickigrzegorz/circular-progress-bar/assets/46477308/8face699-668a-488d-aaef-e681def225ca">

This is chrome screenshot

<img width="268" alt="Screenshot 2024-02-03 230930" src="https://github.com/tomickigrzegorz/circular-progress-bar/assets/46477308/d0903f92-451b-4df4-8807-bf042d704711">

This is screenshot from video. Please see the differnce.

**Comment by tomickigrzegorz** [comment]
And what should I do with this information?
I don't have all devices and browsers, I test what I have. It is possible that you have an old version of the browser where stroke-dashoffset does not work or even the code is incorrect.
Unfortunately, there's nothing I can do about it.
My hardware capabilities are limited. As I wrote above, on iPhone and Safari it works for me.
I have the iPhone 8 iOS 16.72 version and I'm testing it.
Unfortunately, apple browsers are worse than "IE" in the old days, they even have simple problems with max-width in css, but that's a story for another ticket :)
To sum up, I have no way to improve it, if you or someone who reads this can, please do so, but unfortunately I cannot do it.

**Comment by tomickigrzegorz** [comment]
Download the branch and check if it is correct now?
[iOS issue](https://github.com/tomickigrzegorz/circular-progress-bar/tree/tomickigrzegorz/issue87)

**Comment by aparnascodex** [comment]
@tomickigrzegorz the issue was partly resolved with the new update. And it broke the inverse option functionality in chrome which was working earlier. So we did workaround with reversing the color options when inverse option is true.

**Comment by tomickigrzegorz** [comment]
I won't make blind corrections again. I asked you to check it and you didn't do it. Therefore, I rolled back the changes and will not fix it (it works fine for me on Safari).
Download the code and correct it yourself.

**Comment by aparnascodex** [comment]
No issues, we did a workaround . Thank you.
